content/updates/2021-06/2-API-date-filter-on-assets.md

```diff
@@ -0,0 +1,23 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# Improved asset synchronisation within your connectors
+::: meta-data type="Improvement" features="Connectivity" available="mid-June" in="EE" link-to-doc="https://api.akeneo.com/documentation/filter.html#by-update-date-2"
+
+When requesting assets via the REST API, you can now ask to retrieve all assets updated on a date range, since a number of days or even, before a given date. This should heavily help you improve the performance of your asset synchronization in your connectors.
```

Comment:
```suggestion
When requesting assets via the REST API, you can now ask to retrieve all assets updated on a date range, since a number of days, or even before a given date. This should heavily help you improve the performance of your asset synchronization in your connectors.
```